common/src/Utilities/Linux/LnxHostSys.cpp

```diff
@@ -157,6 +157,15 @@ void *HostSys::MmapReservePtr(void *base, size_t size)
     return mmap(base, size, PROT_NONE, MAP_PRIVATE | MAP_ANONYMOUS, -1, 0);
 }
 
+#ifdef __APPLE__
```

Comment:
If MmapReservePtr isn't used anywhere else on mac os then maybe you can put the function in else macro here too. Just a minor optimisation.